stdlib/public/Platform/Darwin.swift

```diff
@@ -23,3 +23,9 @@ public let M_PI_2 = Double.pi / 2
 
 @available(swift, deprecated: 3.0, message: "Please use 'Double.pi / 4' or '.pi / 4' to get the value of correct type and avoid casting.")
 public let M_PI_4 = Double.pi / 4
+
+@available(swift, deprecated: 3.0, message: "Please use '2.squareRoot()'.")
+public let M_SQRT2 = 2.squareRoot()
```

Comment:
Ahh. Right. It is quite possible to simply recommend 2 variants in the message...
On the other hand... If we establish a certain usage pattern with these recommendations, people will hopefully start using it everywhere, regardless of `import Platform`.